# Hypothesis Testing


### Hypothesis Testing is a type of statistical analysis in which you put your assumptions about a population parameter to the test. It is used to estimate the relationship between 2 statistical variables.

### Hypothesis Testing Formula

### Z = ( x̅ – μ0 ) / (σ /√n)

### Here, 
### x̅ is the sample mean,
### μ0 is the population mean,
### σ is the standard deviation,
### n is the sample size.

## Null Hypothesis

### The Null Hypothesis is the assumption that the event will not occur. A null hypothesis has no bearing on the study's outcome unless it is rejected.

### H0 is the symbol for it (H-naught)

##  Alternate Hypothesis 

### Alternate Hypothesis is the logical opposite of the null hypothesis. The acceptance of the alternative hypothesis follows the rejection of the null hypothesis. 
### H1 is the symbol for it.

## Examples

### Q1:.Suppose a child psychologist claims that the average time working mothers spend talking to their children is at least 11 minutes per day. You conduct a random sample of 1000 working mothers and find they spend an average of 11.5 minutes per day talking with their children. Assume prior research suggests the population standard deviation is 2.3 minutes.Conduct a test with a level of significance of alpha = 0.05.

### Formulate the Hypothesis

## Null Hypothesis

### Ho=μ≤11 minutes Average time working mothers spend talking to their children

## Alternative Hypothesis

### Ha=μ>11 minutes Average time working mothers spend talking to their children

## Data sample

### x̅ =11.5 minutes

### n =1000

### σ =2.3

### μ=11

### Calculating Z value

In [1]:
import math

In [2]:
Z=(11.5-11)/(2.3/math.sqrt(1000))
Z

6.874516652539955

### Since the calculated z-score (6.87) is greater than the critical z-value (1.645), we reject the null hypothesis.

### Therefore, we have enough evidence to support the psychologist's claim that the average time working mothers spend talking to their children is greater than 11 minutes per day.

## Q2: A coffee shop claims that their average wait time for customers is less than 5 minutes. To test this claim, a sample of 40 customers is taken, and their wait times are recorded. The sample mean wait time is found to be 4.6 minutes with a standard deviation of 0.8 minutes. Perform a hypothesis test at a significance level of 0.05 and determine whether there is enough evidence to support the coffee shop's claim.

### Formulate the Hypothesis

## Null Hypothesis

### Ho=μ≥ 5 minutes average wait time for customers 

## Alternative Hypothesis

### Ha=μ<5 minutes Average wait time for customers

### x̅ =4.6 minutes

### n =40

### σ =0.8 minutes

### μ=5 minutes

In [5]:
import math

In [6]:
S=(4.6-5)/(0.8/math.sqrt(40))
S

-3.162277660168382

Average wait time is less than 5 minutes.